covid_model/ode_builder.py

```diff
@@ -200,7 +184,10 @@ def param_attr_level(self, attr_name):
 
     # check if a cmpt matches a dictionary of attributes
     def does_cmpt_have_attrs(self, cmpt, attrs, is_param_cmpts=False):
-        return all(cmpt[self.param_attr_level(attr_name) if is_param_cmpts else self.attr_level(attr_name)] == attr_val for attr_name, attr_val in attrs.items())
+        return all(
+            cmpt[self.param_attr_level(attr_name) if is_param_cmpts else self.attr_level(attr_name)]
+            in ([attr_val] if isinstance(attr_val, str) else attr_val)
+            for attr_name, attr_val in attrs.items())
```

Comment:
I changed a `==` to an `in`, which means that now you can provide attrs in a format like {"seir": ["I", "A"]} with multiple attributes for one attribute type, in set_params().